### PR TITLE
update find-pi.md (fixes #667)

### LINKED ISSUE #667: The link for brew installation/tutorial is missing under "Step 1. nmap method"
### Problem
The install/tutorial link for **Homebrew** is missing under the **nmap method** for **Finding Pi**.

 ### Steps to reproduce the problem
1. Navigate to the link https://treehouses.io/#!./pages/vi/find-pi.md
2. Under the section "Method 3: nmap command", the note, 
"NOTE: If this is your first time using the CLI on macOS you need to install brew, a package management system for the CLI"
3. The above note is missing the link for brew.

 ### Screenshots
image

 ### Proposed solution
An install/tutorial link is necessary to be added to the note here, as this is technically the initial step where an intern would have to encounter **brew**.

### PR DESCRIPTION
**Pull request by Pavivenkatesan**
Fixes #667 

### Description
Added brew install link to the section and fixed it

### Raw.Githack preview link
https://raw.githack.com/pavivenkatesan/pavivenkatesan.github.io/brewlink_fix/#!./pages/vi/find-pi.md

### CODE REVIEW

**Review by jayala-29 — APPROVED** [review]
nice addition :)

File: pages/vi/find-pi.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ The `nmap` command ([Network Mapper](https://nmap.org)) is a free and open-sourc
 - macOS: in your terminal, run `brew install nmap`
 - Windows: see the [nmap download page](http://nmap.org/download.html)
 
-**NOTE:** **If this is your first time using the CLI on macOS you need to install brew, a package management system for the CLI**
+**NOTE:** **If this is your first time using the CLI on macOS you need to install [brew](https://brew.sh/),a package management system for the CLI**
```
Comment (fankaiyuan) [inline pages/vi/find-pi.md]: please add a space after the comma

File: pages/vi/gitrepositories.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ The **upstream repository** is the one we are contributing to.
 3. `git remote add upstream https://github.com/treehouses/treehouses.github.io.git`
 4. `git remote -v` origins should remain the same, but you should also be fetching and pushing to OLE as the upstream now
 
-#### Sync Your Fork
+#### Sync/Update Your Fork
```
Comment (fankaiyuan) [inline pages/vi/gitrepositories.md]: I think you have solved this in the another PR, remove this so one RP for one issue

**Review by fankaiyuan — CHANGES_REQUESTED** [review]

File: pages/vi/gitrepositories.md (hunk 2 above)
Comment (Pavivenkatesan) [reply pages/vi/gitrepositories.md]: Done

**Review by fankaiyuan — APPROVED** [review]

**Review by dogi — APPROVED** [review]

**Pull request merged**